src/client.cpp

```diff
@@ -323,45 +315,45 @@ PVR_ERROR GetChannels(ADDON_HANDLE handle, bool bRadio)
 
 bool OpenLiveStream(const PVR_CHANNEL &channel)
 {
-  XBMC->Log(LOG_DEBUG, "%s - OpenLiveStream", __FUNCTION__);
-  if (m_data)
-  {
-    CloseLiveStream();
-
-    if (m_data->GetChannel(channel, m_currentChannel))
-    {
-      m_bIsPlaying = true;
-      m_data->SetPlaying(true);
-      return true;
-    }
-  }
-
   return false;
 }
 
 void CloseLiveStream(void)
 {
-  if (m_data)
-    m_data->SetPlaying(false);
+}
 
-  m_bIsPlaying = false;
+bool SwitchChannel(const PVR_CHANNEL &channel)
+{
+  return false;
 }
 
-int GetCurrentClientChannel(void)
+PVR_ERROR GetStreamProperties(PVR_STREAM_PROPERTIES* pProperties)
 {
-  return m_currentChannel.iUniqueId;
+  return PVR_ERROR_NOT_IMPLEMENTED;
 }
 
-bool SwitchChannel(const PVR_CHANNEL &channel)
+int ReadLiveStream(unsigned char *pBuffer, unsigned int iBufferSize)
 {
-  CloseLiveStream();
+  return -1;
+}
 
-  return OpenLiveStream(channel);
+long long SeekLiveStream(long long iPosition, int iWhence /* = SEEK_SET */)
+{
+  return -1;
 }
 
-PVR_ERROR GetStreamProperties(PVR_STREAM_PROPERTIES* pProperties)
+long long PositionLiveStream(void)
+{
+  return -1;
+}
+
+long long LengthLiveStream(void)
+{
+  return -1;
+}
+
+void PauseStream(bool bPaused)
 {
-  return PVR_ERROR_NOT_IMPLEMENTED;
 }
 
```

Comment:
Proč došlo k přesunu těchto funkcí ze sekce /** UNUSED API FUNCTIONS */? Zbytečná změna.

Reply:
Ano, su tam v client.cpp niektore zmeny ako pozostatok pokusov omylov, ked som sa roznymi sposobmi snazil rozchodit to prehravanie z historie... mozem sa na to este pozriet a dat spat nepotrebne zmeny.

Reply:
Upravil som ten commit (popresuval nepouzivane funkcie spat)...